.ci/testruns/extensions-controllers.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: testmachinery.sapcloud.io/v1beta1
+kind: Testrun
+metadata:
+  generateName: tm-gardener-extensions-controllers-
+  namespace: default
+spec:
+  ttlSecondsAfterFinished: 3600 # 1 hour
```

Comment:
This is the time to live of the testrun resource which means that the CR will be deleted from the testmachinery cluster after 1 hour.
The testmachinery dashboard uses this resource to show the test results which means that after this 1 hour you are unable to view the test results via the testmachinery dashboard.
Therefore, I would extend the ttl so that developers are able to see the test results for more than 1 hour after the test has finished.

Reply:
I see, thanks, done.